frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -252,7 +252,9 @@ function BookInfo:onShowBookCover(file)
                 text = _("No cover image available."),
             })
         end
-        document:close()
+        if not self.document then
```

Comment:
There's already an is_doc for that.

Reply:
Ah, it's local to the other function. But in any case, I'd rather have it more greppable. :-)